gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-creation-workflow.spec.ts

```diff
@@ -92,9 +92,8 @@ describe('API creation workflow', () => {
     });
 
     it('should successfully connect to created API', function () {
-      cy.request(`${Cypress.env('gatewayServer')}/${apiPath}`).then((response) => {
-        expect(response.status).to.eq(200);
-        expect(response.body.message).to.eq('Hello, World!');
+      cy.callGateway(`${Cypress.env('gatewayServer')}/${apiPath}`).then((response) => {
```

Comment:
As the function is called `callGateway` what about using the cypress env var `gatewayServer` directly in the function.
We would only need to provide the path to call instead of building the whole url 🤔

Reply:
True, that sounds more than reasonable 👍🏻 
I'll do that.
Thanks for the hint!